### PR TITLE
OCPBUGS#31451-ocp-13: Added note about vSphere instance versions with…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OCPBUGS-31451

Link to docs preview:
* [VMware vSphere infrastructure requirements](https://74057--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned)
* [VMware vSphere CSI Driver Operator requirements](https://74057--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere#vsphere-csi-driver-reqs_persistent-storage-csi-vsphere)

[x] SME has approved this change.
[x] QE has approved this change.

Additional information:
* https://docs.google.com/document/d/1Y6aCc-F4Wc11sdJtwqL-DYKCQKCY_xJs68tEh8S3v5c/edit

### CODE REVIEW

**Comment by gcharot** [comment]
LGTM

File: installing/installing_vsphere/installing-restricted-networks-vsphere.adoc [hunk 1]
```diff
@@ -41,7 +41,12 @@ include::modules/installation-about-restricted-network.adoc[leveloffset=+1]
 
 include::modules/cluster-entitlements.adoc[leveloffset=+1]
 
-include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
+include::modules/
```
Comment (mburke5678) [inline installing/installing_vsphere/installing-restricted-networks-vsphere.adoc:44]:
```suggestion
```

File: installing/installing_vsphere/installing-restricted-networks-vsphere.adoc [hunk 2]
```diff
@@ -41,7 +41,12 @@ include::modules/installation-about-restricted-network.adoc[leveloffset=+1]
...
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about CSI automatic migration, see "Overview" in xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[VMware vSphere CSI Driver Operator]..adoc[leveloffset=+1]
```
Comment (mburke5678) [inline installing/installing_vsphere/installing-restricted-networks-vsphere.adoc:49]:
```suggestion
* For more information about CSI automatic migration, see "Overview" in xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[VMware vSphere CSI Driver Operator]
```

**Comment by mburke5678** [comment]
@dfitzmau A few nits and one question about a missing module. Otherwise LGTM

File: installing/installing_vsphere/installing-restricted-networks-vsphere.adoc (hunk 2 above)
Comment (dfitzmau) [reply installing/installing_vsphere/installing-restricted-networks-vsphere.adoc:49]: Yeah, thanks for spotting this. An awful copy and paste for sure. Fixed!

File: installing/installing_vsphere/installing-restricted-networks-vsphere.adoc (hunk 1 above)
Comment (dfitzmau) [reply installing/installing_vsphere/installing-restricted-networks-vsphere.adoc]: Slippy fingers must have caused this. Apologies. Fixed by readding the include text.

**Comment by duanwei33** [comment]
LGTM (from storage QE side)

**Pull request merged**